Add basic documentation on projects (#1589)
nextcloud/documentation: user_manual/files/projects.rst

> =======================

Reviewer: ```suggestion
========
```

Projects

> =======================

Reviewer: ```suggestion
========
```

> Users can add files, chats and other items to projects. The various apps will present these items in a list, allowing users to immediately jump to them. Projects are Nextcloud wide, that is, when a user shares files that are part of a project, the other user can see the project with the shared files. A click on any of the items in a project leads right to it, be it a chat, a file or a task.

Reviewer 2: ```suggestion
Users can associate files, chats and other items with each other in projects. The various apps will present these items in a list, allowing users to immediately jump to them. Projects are Nextcloud wide. When a user shares a file that is part of a project, the share recipient can see that project, too. A click on any of the items in a project leads right to it, be it a chat, a file or a task.
```

Reviewer 2: resolved by https://github.com/nextcloud/documentation/commit/3c026c8d06353d0915e9dc048e655263d72476f0

Create a new project
-------------------------------

A new project can be created by linking two items together. Start off by opening a file or folders sharing sidebar. 

.. figure:: ../images/projects1.png

> After clicking the *Add to a project* section and selecting the type of item you want to link with the current file/folder a selector will open that allows you selecting a Talk conversation for example.

Reviewer 2: ```suggestion
Click *Add to a project* select the type of item you want to link with the current file/folder. A selector will open that allows you selecting a Talk conversation for example.
```
shorter sentences

.. figure:: ../images/projects2.png

Once the item has been selected a new project is being created and listed in the sharing tab of the sidebar. The same project will also appear in the sharing sidebar of the linked items.

.. figure:: ../images/projects3.png

The list entry shows quick links to a limited number of items. By opening the context menu, the project can be renamed and the full list of items can be expanded.

Adding more entries to a project
----------------------------------------------

If another item should be added to an already existing project this can be done by searching for the project name in the *Add to a project* picker.

.. figure:: ../images/projects4.png

Visibility of projects
----------------------

> Projects do not influence the access to the different items. Users will only see projects of other users if they have access to all containing items.

Reviewer: ```suggestion
Projects do not influence the access and visibility of the different items. Users will only see projects of other users if they have access to all contained items.
```
